icons/trending-up-down.svg

```diff
@@ -0,0 +1,16 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="m15 15 6 6h-4.8" />
+  <path d="M16.2 3H21l-9 9-4.5-4.5L3 12" />
+  <path d="M21 21v-4.8" />
+  <path d="M21 3v4.8" />
```

Comment:
Let's improve pixel perfection and keep the arrow heads as single elements¹:

```suggestion
  <path d="M14.828 14.828 21 21" />
  <path d="M21 16v5h-5" />
  <path d="m21 3-9 9-4-4-6 6" />
  <path d="M21 8V3h-5" />
```

¹
<img width="145" alt="image" src="https://github.com/user-attachments/assets/08001bce-ff16-4591-ad9a-af98385d2f59">
vs as they currently are:
<img width="165" alt="image" src="https://github.com/user-attachments/assets/0bcecfe3-57cd-471f-b5df-c2fbea3daefb">